internal/topic/topicreaderinternal/stream_reconnector_test.go

```diff
@@ -130,7 +132,10 @@ func TestTopicReaderReconnectorReadMessageBatch(t *testing.T) {
 	})
 
 	xtest.TestManyTimesWithName(t, "OnClose", func(t testing.TB) {
-		reconnector := &readerReconnector{tracer: &trace.Topic{}}
+		reconnector := &readerReconnector{
+			tracer:     &trace.Topic{},
+			initDoneCh: make(empty.Chan),
```

Comment:
the chan must be inited with initChannelsAndClock

Reply:
in that case initChannelsAndClock() shouldn't be used,because other channels won't be closed